From 0884c49b0ffad62641b958b7c8822dfe6adb831c Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Mon, 20 Feb 2023 13:42:55 +0200
Subject: [PATCH] [SW-123737] gaudi2: ext_rom: fix bug in ppboot_secure_arc

The ppboot_secure_arc was not setting the priveledge level
to arcs because the security file was not getting compiled
for erom pcie image and the weak definition of was getting
executed. This patch fixes this by including the reset file
for erom pcie image.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: I81aeb3bddb7701fcd0f399442765a6750637e461
---

diff --git a/app/ext_rom/Makefile b/app/ext_rom/Makefile
index ac46f28..177c279 100644
--- a/app/ext_rom/Makefile
+++ b/app/ext_rom/Makefile
@@ -63,6 +63,7 @@
 	$(HL_DEV_ARCH)/../common/$(HL_TARGET).o \
 	$(HL_DEV_ARCH)/../common/common.o \
 	$(HL_DEV_ARCH)/common.o \
+	$(HL_DEV_ARCH)/../common/security.o \
 	erom_secondary.o
 
 EROM_RECOVERY_DEPS := \
